client/me/connected-application-item/index.jsx

```diff
@@ -60,19 +61,18 @@ module.exports = React.createClass( {
 	},
 
 	renderAccessScopeBadge: function() {
-		var meta = '',
-			connection = this.props.connection;
+		var meta = '';
```

Comment:
I kind of like having the variable shorthand here as the array of `this.props.` seems distracting. A more elegant solution could be taken with destructuring, however:

```js
const { connection: { scope, site } } = this.props;

if ( 'auth' == scope ) …

meta = site.site_name;
```